apps/api/src/app/modules/project/entities/release.entity.ts

```diff
@@ -15,13 +15,13 @@ export class Release extends BaseEntityTemplate{
   @Column()
   version: string;
 
-  @Column()
+  @Column( { nullable: true } )
   hoursBudget: number;
 
-  @Column()
+  @Column( { nullable: true } )
   billableHoursBudget: number;
 
-  @Column()
+  @Column( { nullable: true } )
   expirationDate: Date;
```

Comment:
```suggestion
  deadline: Date;
```
Controllare anche dto se presenti